wdio.conf.js

```diff
@@ -513,10 +513,6 @@ exports.config = {
         '*.ciscospark.com'
       ],
       tunnelDomains: [
-        'whistler-prod.onint.ciscospark.com',
-        'whistler.onint.ciscospark.com',
-        'internal-testing-services.wbx2.com',
-        'calendar-whistler.onint.ciscospark.com',
```

Comment:
Per a discussion, it seems as though these were mostly used for jenkins and should be fine to remove now that we're on circleCi.

Reply:
This is also a "fix" for the tunnelDomains email from saucelabs & a cleanup to be in parity with the widgets who only supply `127.0.0.1, localhost`